wp-sentry.php

```diff
@@ -19,7 +19,7 @@
 }
 
 // Make sure the PHP version is at least 7.1.
-if ( ! defined( 'PHP_VERSION_ID' ) && PHP_VERSION_ID < 70100 ) {
+if ( defined( 'PHP_VERSION_ID' ) && PHP_VERSION_ID < 70100 ) {
```

Comment:
Interesting, this is indeed not doing what it should, what do you think about this suggestion though?

```suggestion
if ( ! defined( 'PHP_VERSION_ID' ) || PHP_VERSION_ID < 70100 ) {
```

According to the [PHP docs](https://www.php.net/manual/en/reserved.constants.php#reserved.constants.core) this constant was introduced in PHP 5.2.7 (and WordPress even has a [polyfill](https://github.com/WordPress/WordPress/blob/5ce95b7c8801764de85c6c8425a758d5b50912e2/wp-includes/random_compat/random.php#L32-L42) for it), however, you never know...

Reply:
Done.